src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -896,9 +896,9 @@ public function onDelete(Closure $callback)
      * @param  string  $name
      * @return bool
      */
-    public function hasScope($name)
+    public function hasNamedScope($name)
```

Comment:
You are renaming public method. Maybe we should add new method, but keep old method for BC with `@deprecated` tag?

Reply:
Hasn't been tagged yet @antonkomarev ... was just merged today.